tsconfig.json

```diff
@@ -9,10 +9,5 @@
     "inlineSources": true,
     "strict": true,
     "skipLibCheck": true
-  },
-  "ts-node": {
-    "transpileOnly": true,
```

Comment:
Just curious why this is getting removed here :) (In case you want fast transpile for development, there is also the `--swc` option for `ts-node` https://typestrong.org/ts-node/docs/swc/)

Reply:
Well, I wasn't using ts-node at all.
I'm just starting a `npx tsc --watch` and use `node src/index.js --cwd test/test-cases/plain` to test my changes.

Feel free to setup tsconfig.json just the way you like :smile: